dev-util/cookiecutter/cookiecutter-1.7.2-r1.ebuild

```diff
@@ -14,7 +14,7 @@ SRC_URI="https://github.com/audreyr/cookiecutter/archive/${PV}.tar.gz -> ${P}.ta
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64"
+KEYWORDS="~amd64 ~x86"
```

Comment:
For some reason this doesn't apply cleanly on ::gentoo repo anymore, but I'll add the KW after testing it on x86.

Reply:
Nevermind, now it did after -3 switch.